mods/default/mapgen.lua

```diff
@@ -550,8 +550,8 @@ function default.register_ores()
 		ore            = "default:stone_with_coal",
 		wherein        = "default:stone",
 		clust_scarcity = 24 * 24 * 24,
-		clust_num_ores = 27,
-		clust_size     = 6,
+		clust_num_ores = 243,
```

Comment:
This causes over 200 coal ores in one place at any depth under 0 (tested in game to be certain). This is far too high.

Reply:
Good point. Will change.